Files/MultilingualResources/Files.ru-RU.xlf

```diff
@@ -242,166 +418,6 @@
           <source>Videos</source>
           <target state="new">Видео</target>
         </trans-unit>
-        <trans-unit id="ExtractFilesDialog.Title" translate="yes" xml:space="preserve">
```

Comment:
These strings should not be removed. If you pull the latest from `develop` it should fix the issue.

Reply:
I fix all. I need some type to test application